hadoop-hdds/container-service/pom.xml

```diff
@@ -33,6 +33,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-hdds-common</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-hdds-common</artifactId>
+      <type>test-jar</type>
```

Comment:
Do we need hadoop-hdds-common as a test-jar type for container-service?

Reply:
Yes, AFAIK that makes the test code from `hadoop-hdds-common` available to the other project.